ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -34,9 +35,11 @@ class AlertsView extends React.Component {
   constructor(props) {
     super(props);
     this.validated = this.props.validated;
+    const { frameworks } = this.props;
     this.state = {
       status: this.getDefaultStatus(),
       framework: getFrameworkData(this.props),
+      alertsViewFrameworks: this.extendDropdownOptions(frameworks),
```

Comment:
I agree we need to differentiate between `frameworks` used in `Alerts view` vs `frameworks` in rest of the views.
But let's use a shorter version, without duplicating the name of the parent component.

Maybe something like `frameworkOptions`?

Reply:
OK